notebook/static/services/sessions/session.js

```diff
@@ -259,7 +259,15 @@ define([
     Session.prototype._update_model = function (data) {
         if (data && data.id) {
             this.id = data.id;
-            this.session_url = utils.url_join_encode(this.session_service_url, this.id);
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
Looks like some conflict markers have been committed here. You want to be careful about resolving conflicts before merging.